app/presenters/document_export_presenter.rb

```diff
@@ -12,60 +12,127 @@ def as_json
 
   def editions
     model.editions.map do |edition|
-      edition_associations(edition)
+      {
+        edition: edition,
+        government: edition.government,
+        whitehall_admin_links: resolve_whitehall_admin_links(edition),
+        associations: {
+          attachments: complete_attachments(edition),
+          authors: stripped_authors(edition),
+          edition_organisations: complete_organisations(edition),
+          edition_policies: stripped_edition_policies(edition),
+          images: complete_images(edition),
+          last_author: stripped_last_author(edition),
+          topical_events: strippped_topical_events(edition),
```

Comment:
This has got three `p`s.

Reply:
Ah damn it, too eager with the p key, good spot!